rov/rov.py

```diff
@@ -105,10 +116,12 @@ def update(self):
                 print "Here"
                 print err
                 print "Also here"
+                print err
             except:
-                print sys.exc_info()
-                #print sys.exc_info()
-                pass
+                i = sys.exc_info()
+                print i[0]
+                print i[1]
+                traceback.print_tb(i[2], limit=1, file=sys.stdout)
```

Comment:
`sys.exc_info()` returns a tuple, the third item is the traceback. I wanted to see more clearly what line it was failing on, so this takes care of that.